### PR TITLE
Add search case insensitive  support

### PR DESCRIPTION
**Pull request by SwordFaith**

### CODE REVIEW

File: rest_server/src/models/marketplace_item.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
-const { isNil } = require('lodash');
-const { Op } = require('sequelize');
+const { isNil, toLower } = require('lodash');
```
Comment (debuggy) [inline rest_server/src/models/marketplace_item.js:3]: where is this func used?

Comment (SwordFaith) [reply rest_server/src/models/marketplace_item.js:3]: Its used to create lower case keyword

File: rest_server/src/models/marketplace_item.js [hunk 2]
```diff
@@ -64,10 +64,29 @@ class MarketplaceItem {
...
+        const lowerKeyword = toLower(keyword);
         filterStatement[Op.or] = [
-          { name: { [Op.substring]: keyword } },
-          { author: { [Op.substring]: keyword } },
-          { summary: { [Op.substring]: keyword } },
+          {
+            name: where(
+              fn('LOWER', col('name')),
+              Op.substring,
+              '%' + lowerKeyword + '%',
```
Comment (debuggy) [inline rest_server/src/models/marketplace_item.js:73]: better use template string 
```
`%${lowerkeyword)%`
```

Comment (SwordFaith) [reply rest_server/src/models/marketplace_item.js:73]: Ok, I'll update this.

**Review by debuggy — APPROVED** [review]

**Pull request merged**